qt_ui/windows/groundobject/QGroundObjectMenu.py

```diff
@@ -133,19 +117,19 @@ def doLayout(self):
         j = 0
         total_income = 0
         received_income = 0
-        for i, building in enumerate(self.buildings):
-            if building.dcs_identifier not in FORTIFICATION_BUILDINGS:
+        for building in self.ground_object.units:
+            if building not in FORTIFICATION_BUILDINGS:
                 self.buildingsLayout.addWidget(
                     QBuildingInfo(building, self.ground_object), j / 3, j % 3
                 )
                 j = j + 1
 
-            if building.category in REWARDS.keys():
-                total_income = total_income + REWARDS[building.category]
-                if not building.is_dead:
-                    received_income = received_income + REWARDS[building.category]
+            if self.ground_object.category in REWARDS.keys():
+                total_income += REWARDS[self.ground_object.category]
+                if building.alive:
+                    received_income += REWARDS[self.ground_object.category]
             else:
-                logging.warning(building.category + " not in REWARDS")
+                logging.warning(self.ground_object.category + " not in REWARDS")
```

Comment:
This generates a warning for other ground objects as well. should really only check buildings or better just statics